src/main/java/com/mitchej123/hodgepodge/mixins/late/thaumcraft/MixinUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.mitchej123.hodgepodge.mixins.late.thaumcraft;
+
+import net.minecraft.item.Item;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import cpw.mods.fml.common.Loader;
+
+@Mixin(thaumcraft.common.lib.utils.Utils.class)
+public class MixinUtils {
+
+    @Inject(method = "isEETransmutionItem", at = @At("HEAD"), remap = false, cancellable = true)
```

Comment:
this method is called thousands of times, could you instead overwrite it and cache the class if it's present to only do the ModLoaded + Class.forName once and make this method faster